oauth2_http/java/com/google/auth/oauth2/ExternalAccountMetricsHandler.java

```diff
@@ -43,7 +43,7 @@ class ExternalAccountMetricsHandler implements java.io.Serializable {
 
   private final boolean configLifetime;
   private final boolean saImpersonation;
-  private String credentialSourceType;
+  private ExternalAccountCredentials credentials;
```

Comment:
This needed to change so I could keep the metrics handler final in External account credentials. Since the metric source could change due to things in child credential specific builder and we create the metrics handler in the parent class constructor.